app/src/test/java/io/intrepid/contest/screens/contestoverview/ContestOverviewPresenterTest.java

```diff
@@ -58,6 +58,36 @@ public void onViewCreatedShouldShowContestNameWhenSuccessfullyRetrievedContestDe
     }
 
     @Test
+    public void onViewCreatedShouldShowRatingGuide() throws Exception {
+        setupSuccessfulContestDetailsCall();
+
+        presenter.onViewCreated();
+        testConfiguration.triggerRxSchedulers();
+
+        verify(mockView).showRatingGuide();
+    }
+
+    @Test
+    public void onViewCreatedShouldShowContestDescriptionWhenSuccessfullyRetrievedContestDetails() throws Exception {
```

Comment:
While i like the descriptive method names, i think that this is way overkill. Perhaps rename to just `showContestDescription`